src/fsharp/FSharp.Core/SR.fs

```diff
@@ -26,13 +26,11 @@ module internal SR =
     let inputMustBeNonNegative = "inputMustBeNonNegative"
     let inputMustBePositive = "inputMustBePositive"
     let enumerationPastIntMaxValue = "enumerationPastIntMaxValue"
-    let inputSequenceEmpty = "inputSequenceEmpty"
```

Comment:
best way, next version move everything from `SR` to `Resources`, problem fixed

- `SR.GetString(SR.inputMustBePositive)` -> `Resources.inputMustBePositive()`
- "inputMustBePositive" only inside `Resources.inputMustBePositive`, no need for SR.Resources.inputMustBePositive
- remove `SR` module